.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+### Description
+
+[Description of the bug or feature]
+
+### Steps to Reproduce
+
+1. [First Step]
+2. [Second Step]
+3. [and so on...]
+
+### Expected behavior:
+
+[What you expected to happen]
+
+### Actual behavior:
+
+[What actually happened]
+
+### Versions
+
+[You can get this information from executing `npm list --depth=0`, copy and paste here]
```

Comment:
What about asking for the node (`node -v`) and npm (`npm -v`) versions too?

Reply:
Done.